rules/RULE_5_3_A_provide_doxygen_function_comment_on_function_in_header.py

```diff
@@ -44,13 +44,13 @@ def RunRule(lexer, fullName, decl, contextStack, context) :
     if ext == ".h" :
         upperBlock = contextStack.SigPeek()
         # modification to make doxygen compulsory for even private functions
-        if upperBlock is None:  # != None: and upperBlock.type == "CLASS_BLOCK" and upperBlock.additional == "PRIVATE":
+        if upperBlock is not None and upperBlock.type == "CLASS_BLOCK" and upperBlock.additional == "PRIVATE":

```

Comment:
Test 4 has private function without comments, and the test passes(well, in my environment :P).
I think it would be nice that you can run the tests as double check.